pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -1394,12 +1392,12 @@ benchmarks! {
             total_staking_reward: total_staked,
         });
 
-        let mut nominations: BoundedVec<Bond<T::AccountId, BalanceOf<T>>, MaxNominations> = BoundedVec::default();
+        let mut nominations: Vec<Bond<T::AccountId, BalanceOf<T>>> = Vec::new();
```

Comment:
This is also a part of another pr related to the boundedvec revert, so don't think this should be in your pr